zerver/openapi/zulip.yaml

```diff
@@ -4401,15 +4401,14 @@ paths:
                                         The [policy](/api/roles-and-permissions#permission-levels)
                                         for which users can create bot users in this organization.
                                     can_access_all_users_group:
-                                      type: integer
-                                      description: |
-                                        The ID of the [user group](/api/get-user-groups) whose members
-                                        are allowed to access all users in the organization.
-
-                                        This setting can currently only be set to `"role:everyone"`
-                                        system group.
+                                      allOf:
+                                        - $ref: "#/components/schemas/GroupSettingValue"
+                                        - description: |
+                                            A [group-setting value](/api/group-setting-values) defining the
+                                            set of users who are allowed to access all users in the
+                                            organization.
 
-                                        **Changes**: New in Zulip 8.0 (feature level 225).
+                                            **Changes**: New in Zulip 8.0 (feature level 225).
```

Comment:
This is missing the new changelog entry

Reply:
[Fixed.](https://github.com/zulip/zulip/compare/8628c9625b928c5de53d44b20e39d5789829051a..140c4441397f0a1fc0362b86613fad547b7b4b5f)